tests/integration/test_vulnerability_detector/data/feeds/canonical/custom_feed_updated_packace_still_vulnerable.xml

```diff
@@ -0,0 +1,51 @@
+    xmlns:linux-def="http://oval.mitre.org/XMLSchema/oval-definitions-5#linux"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://oval.mitre.org/XMLSchema/oval-common-5 oval-common-schema.xsd   http://oval.mitre.org/XMLSchema/oval-definitions-5 oval-definitions-schema.xsd   http://oval.mitre.org/XMLSchema/oval-definitions-5#independent independent-definitions-schema.xsd   http://oval.mitre.org/XMLSchema/oval-definitions-5#unix unix-definitions-schema.xsd   http://oval.mitre.org/XMLSchema/oval-definitions-5#macos linux-definitions-schema.xsd">
+
+    <generator>
+        <oval:product_name>Canonical CVE OVAL Generator</oval:product_name>
+        <oval:product_version>1.1</oval:product_version>
+        <oval:schema_version>5.11.1</oval:schema_version>
+        <oval:timestamp>2021-11-16T15:30:28</oval:timestamp>
+    </generator>
+    <definitions>
+        <definition class="vulnerability" id="oval:com.ubuntu.focal:def:1" version="1">
+            <metadata>
+                <title>CVE-2023 custom-package-0</title>
+                <affected family="unix">
+                    <platform>Ubuntu 22.04 LTS</platform>
+                    <platform>Ubuntu 20.04 LTS</platform>
+                    <platform>Ubuntu 18.04 LTS</platform>
+                    <platform>Ubuntu 16.04 LTS</platform>
+                    <platform>Ubuntu 14.04 LTS</platform>
+                </affected>
+                <reference source="CVE" ref_id="CVE-2023" ref_url="https://github.com/wazuh/wazuh-qa" />
+            </metadata>
+            <criteria>
+                <criteria operator="OR">
+                    <criterion test_ref="oval:com.ubuntu.focal:tst:1" comment="custom-package-0 package in focal, is related to the CVE in some way and has been fixed (note: '10.0.0')." />
+                </criteria>
+            </criteria>
+        </definition>
+    </definitions>
+    <tests>
+        <linux-def:dpkginfo_test id="oval:com.ubuntu.focal:tst:1" version="1" check_existence="at_least_one_exists" check="at least one" comment="Does the 'custom-package-0' package exist and is the version less than '10.0.0'?">
+            <linux-def:object object_ref="oval:com.ubuntu.focal:obj:1"/>
+            <linux-def:state state_ref="oval:com.ubuntu.focal:ste:1" />
+        </linux-def:dpkginfo_test>
+    </tests>
+    <objects>
+        <linux-def:dpkginfo_object id="oval:com.ubuntu.focal:obj:1" version="1" comment="The 'custom-package-0' package binary.">
+            <linux-def:name>custom-package-0</linux-def:name>
+        </linux-def:dpkginfo_object>
+    </objects>
+    <states>
+        <linux-def:dpkginfo_state id="oval:com.ubuntu.focal:ste:1" version="1" comment="The package version is less than '10.0.0'.">
+            <linux-def:evr datatype="debian_evr_string" operation="less than">10.0.0</linux-def:evr>
+        </linux-def:dpkginfo_state>
+    </states>
+</oval_definitions>
```

Comment:
Add end line

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4389/commits/308bc8c4b492b99c117039f889eef100fbdd668d